solaris/solaris11/SPECS/template_agent_v3.12.0.json

```diff
@@ -0,0 +1,1635 @@
+{
```

Comment:
Check if this template is right, please.

Reply:
Done in 7f378b0